documentation/asciidoc/topics/proc_creating_service_monitor.adoc

```diff
@@ -1,32 +1,22 @@
-[id='creating_service_monitor-{context}']
+[id='creating-service-monitor_{context}']
 = Creating a Prometheus Service Monitor
-Define a service monitor instances that configures Prometheus to monitor your
-{brandname} cluster.
+Define service monitor instances that configures Prometheus to monitor your {brandname} cluster.
 
 .Prerequisites
 
+* Enable monitoring for your {openshiftshort} project.
 * Set up a Prometheus stack on your {k8s} cluster.
+* Retrieve user credentials.
```

Comment:
If I'm not wrong, this should be admin credentials from the `<cr-name>-generated-operator-secret` and username should be `operator`.

Reply:
We also need to provide a CLI script example for extracting username and password from the our Secret format

Reply:
@dmvolod Like this one: https://github.com/infinispan/infinispan-operator/blob/master/documentation/asciidoc/topics/cmd_examples/oc_get_secret_creds_jp.adoc ?

Reply:
Yes, @oraNod ,right.